gitlabform/gitlabform/test/test_group_secret_variables.py

```diff
@@ -8,7 +8,7 @@
 GROUP_AND_PROJECT_NAME = GROUP_NAME + '/' + PROJECT_NAME
 
 
-@pytest.fixture(scope="function")
+@pytest.fixture(scope="module")
```

Comment:
This change breaks the tests in this class. After reverting it the tests pass on my machine.